src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,39 @@
+package roomescape.controller;
+
+import java.util.List;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import roomescape.dto.ReservationRequestDto;
+import roomescape.dto.ReservationResponseDto;
+import roomescape.service.ReservationService;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private final ReservationService reservationService;
+
+    public ReservationController() {
+        this.reservationService = new ReservationService();
+    }
+
+    @GetMapping()
```

Comment:
어노테이션에 아무 인자도 없다면 괄호를 생략합니다.
```suggestion
    @GetMapping
```